### PR TITLE
Feat/revert coinjoin onboarding modals

### LINKED ISSUE #6475: CoinJoin onboarding modals
Follow-up of https://github.com/trezor/trezor-suite/issues/6212
<details open>
<summary>1. Account info</summary>
<br/>

</details>

<details open>
<summary>2. Setting desired anonymity level</summary>
<br/>

- Reuse: https://github.com/trezor/trezor-suite/issues/6116
- Warning in Custom settings of anonymity level appears only when user selects anonymity level 5 or lower.

| Group 18800 |  |
| --- | --- |
</details>

### PR DESCRIPTION
**Pull request by matejkriz**
## Description

Reverting CoinJoin onboarding modals so the user is not forced to go through it again and again (and we don't have to implement selection of anonymity level before the account is created).

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/6475

## Screenshots (if appropriate):
image

### CODE REVIEW

File: packages/suite/src/support/messages.ts [hunk 1]
```diff
@@ -3833,35 +3833,6 @@ export default defineMessages({
...
-    },
-    TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_DESCRIPTION: {
-        id: 'TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_DESCRIPTION',
-        defaultMessage:
-            'Inputs from multiple senders are combined into a single transaction, making it difficult for external parties to analyze your connection to an old address.',
-    },
-    TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_FEE_MESSAGE: {
-        id: 'TR_COINJOIN_ACCESS_ACCOUNT_STEP_INITIAL_FEE_MESSAGE',
-        defaultMessage: 'A fee of {fee}% of the used amount is applied',
-    },
```
Comment (karliatto) [inline packages/suite/src/support/messages.ts]: It looks like those messages are still used in component `packages/suite/src/views/wallet/transactions/components/CoinjoinAccountEmpty.tsx`.

File: packages/suite/src/components/suite/modals/AddAccount/components/AddCoinJoinAccountButton.tsx [hunk 2]
```diff
@@ -88,6 +81,7 @@ export const AddCoinJoinAccountButton = ({ network }: AddCoinJoinAccountProps) =
                 isDisabled ? <Translation id="MODAL_ADD_ACCOUNT_LIMIT_EXCEEDED" /> : null
             }
             handleClick={onCreateCoinjoinAccountClick}
+            isLoading={isLoading}
```
Comment (karliatto) [inline packages/suite/src/components/suite/modals/AddAccount/components/AddCoinJoinAccountButton.tsx:84]: Thanks! That loading effect was really needed. :+1:

**Review by karliatto — CHANGES_REQUESTED** [review]
It looks like an improvement in simplification for the user!
Just added a comment to some deleted messages that are still used in other component.

**Comment by dahaca** [comment]
So they will be disabled for MVP? The modals had some important explanations :(

File: packages/suite/src/support/messages.ts (hunk 1 above)
Comment (matejkriz) [reply packages/suite/src/support/messages.ts]: thanks for noticing, I was resolving the conflicts before merging empty account page...

File: packages/suite/src/components/suite/modals/AddAccount/components/AddCoinJoinAccountButton.tsx (hunk 2 above)
Comment (matejkriz) [reply packages/suite/src/components/suite/modals/AddAccount/components/AddCoinJoinAccountButton.tsx:84]: Your idea 😉 still not perfect, there is a period with modal already closed but CoinJoin account still not visible...

**Comment by matejkriz** [comment]
> So they will be disabled for MVP? The modals had some important explanations :(

Yes, that is the plan https://github.com/trezor/trezor-suite/issues/6475#issuecomment-1290737312

What explanations you found important there? Aren't they on that new empty account page?

File: packages/suite/src/support/messages.ts (hunk 1 above)
Comment (matejkriz) [reply packages/suite/src/support/messages.ts]: forcepushed, keeping the messages used on CoinjoinAccountEmpty

**Review by karliatto — APPROVED** [review]

**Pull request merged**